### DISCUSSION TITLE
[FR]: dialog函数式使用时如何获取内部组件的值?

### DISCUSSION THREAD

**Discussion by AnxysUaen** [jdf2e/nutui#2967] [Q&A]
### NutUI 包名

@nutui/nutui

### 这个功能解决了什么问题？

dialog函数式使用时,内部嵌入其他组件,并进行选择或者填写信息等操作,此时希望可以获取子组件的值

### 你期望的组件设计是怎样的？

是否可以在onOk回调内调用子组件中暴露的特定名称的方法,取返回值加入onOk的入参?

**Comment by eiinu** [comment]
1、如果是想在点击确认时，做一些表单校验操作，可以通过 `ok-auto-close` 属性拦截确认按钮的点击关闭操作，自行处理 Dialog 的展示。
2、函数式调用只适合简单的内容，复杂的 Dialog 建议用标签式的方式。
